### ISSUE TITLE
inputmask errors

### ISSUE THREAD

**Issue by gh-supersudo** [RobinHerbots/Inputmask#2370]
I'm using v3.3.11.

I use the jquery.inputmask.bundle.js

When I import it:
```
import inputmask from "inputmask/dist/jquery.inputmask.bundle";
```

I'm getting these errors: `ReferenceError: jQuery is not defined` and `ReferenceError: $ is not defined`

Event though I already have imported jquery before inputmask
```
window.$ = window.jQuery = require('jquery');
```

and I'm getting `$(...).inputmask is not a function` when I just import inputmask like this:
```
import Inputmask from "inputmask";
```

Is there a way to fix this?

**Comment by RobinHerbots** [comment]
@gh-supersudo , 

Can you use the latest version and import inputmask/dist/jquery.inputmask

**Comment by gh-supersudo** [comment]
@RobinHerbots 

I can't because of this bug: https://github.com/RobinHerbots/Inputmask/issues/1922

**Comment by RobinHerbots** [comment]
@gh-supersudo , 

Issue #1922 is fixed. inputmask@5.0.6-beta.3

**Comment by cdmjr** [comment]
@RobinHerbots 

I found a bug, here is the pen: https://codepen.io/Supersudo/pen/BaKoJYw
I have set the min to "01-01-1950", the 02/3 bug didn't happen but when I type the date to `1111101`, it sets to `11-11-1909` which is incorrect.

**Comment by RobinHerbots** [comment]
@cdmjr ,

You are correct, I will fix that tomorrow!

**Comment by RobinHerbots** [comment]
@cdmjr , 

Ah, I already had fixed that in version inputmask@5.0.6-beta.4

**Comment by gh-supersudo** [comment]
Thanks for the help! I'll close the issue now.
